lazyflow/request/threadPool.py

```diff
@@ -29,32 +30,23 @@
 logger = logging.getLogger(__name__)
 
 
-from lazyflow.utility.priorityQueue import PriorityQueue
-
-
 class ThreadPool:
     """
     Manages a set of worker threads and dispatches tasks to them.
     """
 
-    # _DefaultQueueType = FifoQueue
-    # _DefaultQueueType = LifoQueue
-    _DefaultQueueType = PriorityQueue
-
-    def __init__(self, num_workers, queue_type=_DefaultQueueType):
+    def __init__(self, num_workers):
         """
         Constructor.  Starts all workers.
 
         :param num_workers: The number of worker threads to create.
-        :param queue_type: The type of queue to use for prioritizing tasks.  Possible queue types include :py:class:`PriorityQueue`,
-                           :py:class:`FifoQueue`, and :py:class:`LifoQueue`, or any class with ``push()``, ``pop()``, and ``__len__()`` methods.
         """
         self.job_condition = threading.Condition()
```

Comment:
not needed?!

Reply:
A leftover from rebasing to master, will fix that.